transactions.sql

```diff
@@ -0,0 +1,39 @@
+-- update the animals table by setting the species column to unspecified.
+-- Then roll back the change.
+
+BEGIN;
+update animals set species = 'unspecified';
+ROLLBACK;
```

Comment:
Please add a query to verify explicitly that change was made before and after rollback.